config/manifests/bases/opendatahub-operator.clusterserviceversion.yaml

```diff
@@ -23,6 +23,11 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
+    - description: Auth is the Schema for the auths API
+      displayName: Auth
```

Comment:
We should add auth crd to `operators.operatorframework.io/internal-objects: ` annotation on L12

Reply:
:+1:

Reply:
Maybe we can remove it from annotation when odhdashboardconfig is deprecated?